checkov/common/util/ext_argument_parser.py

```diff
@@ -520,3 +520,9 @@ def add_parser_args(self) -> None:
             default=False,
             help="will scan the history of commits for secrets"
         )
+        self.add(
+            "--secrets-history-timeout",
+            action="append",
```

Comment:
Can it be a list of values? or just a single value?

`action="append"` is useful for cases when you repeat the argument, if it is not needed you can remove this line as the default action is 'store'

https://docs.python.org/3/library/argparse.html#action